### PR TITLE
Add aws cli to build-harness

### PR DESCRIPTION
**Pull request by mkj28**
**Why**
Needed for agent-desktop widget deployments.

Currently part of release-harness, but will move here/upstream.

**Testing**
widgets deployed on master

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,16 @@
 FROM alpine:3.12
 
 ARG KUBE_VERSION="v1.16.10"
+ENV AWS_DEFAULT_REGION us-west-1
+ENV AWSCLI_VERSION 1.14.50
```
Comment (albertrdixon) [inline Dockerfile:5]: not really blocking, but...

awscli v1.14.50 is 3 years old, any reason not to use a more recent version?

**Review by albertrdixon — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,16 @@
 FROM alpine:3.12
 
 ARG KUBE_VERSION="v1.16.10"
+ENV AWS_DEFAULT_REGION us-west-1
```
Comment (albertrdixon) [inline Dockerfile:4]: informational: why `us-west-1` and not `us-west-2` ?

Comment (mkj28) [reply Dockerfile:4]: No reason, making it so

File: Dockerfile (hunk 1 above)
Comment (mkj28) [reply Dockerfile:5]: bumping to 1.19.107

**Pull request merged**